src/Compilers/Server/VBCSCompilerTests/AnalyzerConsistencyCheckerTests.cs

```diff
@@ -3,7 +3,6 @@
 // See the LICENSE file in the project root for more information.
 
 #if NETFRAMEWORK
-#nullable disable
```

Comment:
How did the analyzer decide this was ok to enable? Also... how did it compile without a warning before?

Reply:
Have the same question here. This file has a lot of reference types involved so not sure how it fell into the rules you said above. True this is only in `#if NETFRAMEWORK` so we never run nullable analysis there but still doesn't quite square with how I read the rules above

Reply:
I suspect this was triggered by #70210

Reply:
In this case though the `#if` block is reachable, it's one of the multi-targets that we build for this project. If that is the case though does that mean we have to evaluate all of the removals to see if they occur in `#if NETFRAMEWORK` blocks?

Reply:
I think we only ever enable nullability analysis on netstandard2.0 and .NET.

Reply:
We only disable nullable _warnings_, not annotations. Yes, in this particular case, because this is a test project, no one would notice, but this a public API in a DLL. This would cause nullable annotations to be enabled for the file, which was an unintended change, and the analysis needs to be updated to be fixed for this case.